.azure-pipelines/windows-msbuild.bat

```diff
@@ -16,21 +16,17 @@ set N=3
 set DM_MAKE=%DMD_DIR%\dm\path\make.exe
 set LDC_DIR=%DMD_DIR%\ldc2-%LDC_VERSION%-windows-multilib
 
+if "%D_COMPILER%" == "ldc" set HOST_DMD=%LDC_DIR%\bin\ldmd2.exe
+if "%D_COMPILER%" == "dmd" set HOST_DMD=%DMD_DIR%\dmd2\windows\bin\dmd.exe
+
 REM take the first found cl.exe, in case there was already one in the path when vcvarsall.bat was called
-FOR /F "tokens=* USEBACKQ" %%F IN (`where cl.exe`) DO (SET MSVC_CC=%%~fsF && goto CC_DONE)
+FOR /F "tokens=* USEBACKQ" %%F IN (`where cl.exe`) DO (SET MSVC_CC=%%~fsF
+  goto CC_DONE)
```

Comment:
Previously, `MSVC_CC` included a trailing space, which broke the `usingMicrosoftCompiler` detection in `run.d`...